swan-lake/development-tutorials/source-code-dependencies/style-guide/coding-conventions.md

```diff
@@ -278,4 +278,16 @@ intro: This Ballerina Style Guide aims at maintaining a standard coding style am
     };
     ```
 
+## String templates
```

Comment:
Do we need a separate heading for this? Shouldn't it come under "Indentation and line length"?